ports/tftp-hpa/qnx-sa-restart.patch

```diff
@@ -0,0 +1,15 @@
+diff '--color=auto' -ur tftp-hpa-5.2.orig/lib/bsdsignal.c tftp-hpa-5.2/lib/bsdsignal.c
+--- tftp-hpa-5.2.orig/lib/bsdsignal.c	2024-12-17 09:58:08.647491726 -0500
++++ tftp-hpa-5.2/lib/bsdsignal.c	2024-12-17 09:58:47.523167958 -0500
+@@ -6,6 +6,11 @@
+ 
+ #include "config.h"
+ 
++// QNX hack
++#ifndef SA_RESTART
```

Comment:
would it be better to have this check __QNX__ instead so that we could send these small changes upstream in the future?
```
#ifndef __QNX__
#define SA_RESTART 0
#endif
```

Reply:
Not sure if they will be interested in support QNX though. It's developed on kernel.org.

Reply:
That's a good point - in that case, I think this is good.